README.md

```diff
@@ -78,15 +79,23 @@ Each row represents a fastq file (single-end) or a pair of fastq files (paired e
 > provided by the `-c` Nextflow option can be used to provide any configuration _**except for parameters**_;
 > see [docs](https://nf-co.re/usage/configuration#custom-configuration-files).
 
-Now, you can run the pipeline using:
+Now, you can run the pipeline.
+
+Example run:
 
 ```bash
-nextflow run nf-core/rnaseq \
-    --input <SAMPLESHEET> \
-    --outdir <OUTDIR> \
-    --gtf <GTF> \
-    --fasta <GENOME FASTA> \
-    -profile <docker/singularity/.../institute>
+nextflow run https://github.com/epicrispr-biotechnologies/epic-rnaseq \
+    --user tborrman \
+    --study JIRA_XXXX_study \
+    --input s3://epic-bio-data-us-west-2/nextflow/nextflow_data/off-target-pipeline/RNAseq_1M_samplesheet.csv \
```

Comment:
If it's all the same, I'd rather keep the parameters generic and put these specific paths in some kind of example file